aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -117,6 +120,9 @@ Resources:
       ConnectionDrainingPolicy:
         Enabled: true
         Timeout: 300
+<% unless frontends -%>
+      Instances: [WebServer]]
+<% end -%>
```

Comment:
This adds the daemon server to the load-balancer's instance list when the stack does not have additional frontends.